### PR TITLE
rebrand

### PR DESCRIPTION
**Pull request by explicite**
Rebranding

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,15 +46,15 @@ Run indiviudal test
 
 Run SHC examples
 
-    ./bin/spark-submit --verbose --class org.apache.spark.sql.execution.datasources.hbase.examples.HBaseSource --master yarn-cluster --packages com.hortonworks:shc-core:1.1.1-2.1-s_2.11 --repositories http://repo.hortonworks.com/content/groups/public/ --files /usr/hdp/current/hbase-client/conf/hbase-site.xml shc-examples-1.1.1-2.1-s_2.11-SNAPSHOT.jar
+    ./bin/spark-submit --verbose --class org.apache.spark.sql.execution.datasources.hbase.examples.HBaseSource --master yarn-cluster --packages org.virtuslab:shc-core:1.1.1-2.1-s_2.11 --repositories https://repo1.maven.org/maven2/ --files /usr/hdp/current/hbase-client/conf/hbase-site.xml shc-examples-1.1.1-2.1-s_2.11-SNAPSHOT.jar
```
Comment (btomala) [inline README.md:49]: Do we have this connector in maven2 repository?

File: README.md [hunk 2]
```diff
@@ -46,15 +46,15 @@ Run indiviudal test
...
 
 The following illustrates how to run your application in real hbase cluster. You need to provide the hbase-site.xml. It may subject to change based on your specific cluster configuration.
 
-    ./bin/spark-submit  --class your.application.class --master yarn-client  --packages com.hortonworks:shc-core:1.1.1-2.1-s_2.11 --repositories http://repo.hortonworks.com/content/groups/public/ --files /etc/hbase/conf/hbase-site.xml /To/your/application/jar
+    ./bin/spark-submit  --class your.application.class --master yarn-client  --packages org.virtuslab:shc-core:1.1.1-2.1-s_2.11 --repositories https://repo1.maven.org/maven2/ --files /etc/hbase/conf/hbase-site.xml /To/your/application/jar
```
Comment (btomala) [inline README.md:53]: Shouldn't we change the version if we added a feature?

File: README.md [hunk 3]
```diff
@@ -122,31 +122,31 @@ Given a DataFrame with specified schema, above will create an HBase table with 5
...
+    libraryDependencies += “org.virtuslab/shc-core:1.1.1-2.1-s_2.11”
 
 ## Running in secure cluster
 
 For running in a Kerberos enabled cluster, the user has to include HBase related jars into the classpath as the HBase token
 retrieval and renewal is done by Spark, and is independent of the connector. In other words, the user needs to initiate the
 environment in the normal way, either through kinit or by providing principal/keytab. The following examples show how to run
 in a secure cluster with both yarn-client and yarn-cluster mode. Note that if your Spark does not contain [SPARK-20059](https://github.com/apache/spark/pull/17388),
-which is in Apache Spark 2.1.1+, and [SPARK-21377](https://issues.apache.org/jira/browse/SPARK-21377), which is in Apache Spark 2.3.0+, you need to set SPARK_CLASSPATH for both modes (refer [here](https://github.com/hortonworks-spark/shc/wiki/1.-Set-SPARK_CLASSPATH)).
+which is in Apache Spark 2.1.1+, and [SPARK-21377](https://issues.apache.org/jira/browse/SPARK-21377), which is in Apache Spark 2.3.0+, you need to set SPARK_CLASSPATH for both modes (refer [here](https://github.com/VirtusLab/shc/wiki/1.-Set-SPARK_CLASSPATH)).
```
Comment (btomala) [inline README.md:141]: Wiki doesn't exist.

File: README.md [hunk 4]
```diff
@@ -177,7 +177,7 @@ Also, users need to config principal and keytab as below before running their ap
 
 ## Others
 ### Example. Support of Avro schemas:
-The connector fully supports all the avro schemas. Users can use either a complete record schema or partial field schema as data type in their catalog (refer [here](https://github.com/hortonworks-spark/shc/wiki/2.-Native-Avro-Support) for more detailed information).
+The connector fully supports all the avro schemas. Users can use either a complete record schema or partial field schema as data type in their catalog (refer [here](https://github.com/VirtusLab/shc/wiki/2.-Native-Avro-Support) for more detailed information).
```
Comment (btomala) [inline README.md:180]: Wiki doesn't exist

**Review by btomala — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (explicite) [reply README.md:141]: Created

File: README.md (hunk 4 above)
Comment (explicite) [reply README.md:180]: Created

File: README.md (hunk 1 above)
Comment (explicite) [reply README.md:49]: nor right now. we need to release first

File: README.md (hunk 2 above)
Comment (explicite) [reply README.md:53]: Not right now. We need to release it

**Pull request merged**